apps/api/v1/pages/api/event-types/_post.ts

```diff
@@ -275,24 +275,57 @@ async function postHandler(req: NextApiRequest) {
     children: _,
     ...parsedBody
   } = schemaEventTypeCreateBodyParams.parse(body || {});
+  if (parsedBody.teamId && parsedBody.parentId) {
```

Comment:
if teamId is defined {
do things
} else {
do other things
}

Reply:
Do you think it would be better to modify the request body instead of throwing an error?
```
if (parsedBody.teamId) {
    //delete the params , that cannot be present with teamId
    delete req.body.parentId;
    delete parsedBody.parentId;
    ....
} else {
   //delete other params
}
```

Reply:
not necessarily, I mean it would be great if the whole function was clearly split between the things that are done when we have a teamId and the things that are done when we don't have a teamId.